### PR TITLE
Fix couple of typos

### PR DESCRIPTION
**Pull request by mandoonandy**
Little typos before I forget

### CODE REVIEW

File: golang/example.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ func main() {
 		return
 	}
 
-	fmt.Printf("Fetcehd %s\n", network.Name)
+	fmt.Printf("Fetched %s\n", network.Name)
```
Comment (mikalstill) [inline golang/example.go:124]: I presume this example should move into the golang API client eventually?

File: shakenfist/client/main.py [hunk 2]
```diff
@@ -199,7 +199,7 @@ def network_events(ctx, network_uuid=None):
         filtered_events = []
         for e in events:
             filtered_events.append(filter_dict(
-                n, ['timestamp', 'fqdn', 'operation', 'phase', 'duration', 'message']))
+                e, ['timestamp', 'fqdn', 'operation', 'phase', 'duration', 'message']))
```
Comment (mikalstill) [inline shakenfist/client/main.py:202]: Oh, good catch.

File: shakenfist/client/main.py [hunk 3]
```diff
@@ -488,7 +488,7 @@ def instance_events(ctx, instance_uuid=None):
         for e in events:
             filtered_events.append(filter_dict(
                 e, ['timestamp', 'fqdn', 'operation', 'phase', 'duration', 'message']))
-        print(json.dumps({'networks': filtered_events},
+        print(json.dumps({'events': filtered_events},
```
Comment (mikalstill) [inline shakenfist/client/main.py:491]: Again, nice.

File: golang/example.go (hunk 1 above)
Comment (mandoonandy) [reply golang/example.go:124]: yes. doing that now.

**Pull request merged**